bunner-master/bunner.py

```diff
@@ -122,21 +122,27 @@ def handle_input(self, dir):
     def _ai_decide(self, current_row, next_row):
         if isinstance(current_row, Grass):
             direction = 0
+        
+        if isinstance(next_row, Grass):
+            direction = 0
             
-        if isinstance(current_row, Road): 
+        if isinstance(next_row, Road): 
```

Comment:
Make sure to check for cars or logs if current row is road/ water and there are obstacles on the right/left

Reply:
![CleanShot 2024-07-29 at 11 47 38](https://github.com/user-attachments/assets/dc9bc911-752d-49da-b871-4b123fc144ec)